tools/c7n_gcp/c7n_gcp/policy.py

```diff
@@ -39,17 +39,18 @@ class FunctionMode(ServerlessExecutionMode):
     def __init__(self, policy):
         self.policy = policy
         self.log = logging.getLogger('custodian.gcp.funcexec')
+        self.region = self.policy.data['mode'].get('region', mu.DEFAULT_REGION)
```

Comment:
hmm.. this should be in schema, although I wonder if this shouldn't just use the region on a policy. I'm also a little unclear that this makes sense, as it requires duplicating the policy for every region to deploy. generally we would prefer to do that via the cli.

Reply:
This field is already in schema (`c7n_gcp/policy.py:34`). I didn't use the region which comes from cli (in case specified) because the region field from a policy data is already used for the Cloud Scheduler (`mu.py/PeriodicEvent`). 
I agree with what you say about duplicating so I will fix that by modifying both classes to keep the consistency between cloud function and scheduler.

Reply:
@kapilt done